packages/cozy-harvest-lib/src/components/DisconnectedAccountModal.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import { CozyProvider } from 'cozy-client'
+import { BreakpointsProvider } from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+import I18n from 'cozy-ui/transpiled/react/I18n'
+
+import DisconnectedAccountModal from './DisconnectedAccountModal'
+import bankAccounts from './KonnectorConfiguration/ConfigurationTab/bank-account-fixture.json'
+import en from '../locales/en.json'
+
+describe('DisconnectedAccountModal', () => {
+  const setup = () => {
+    const mockClient = {}
+    const root = render(
+      <CozyProvider client={mockClient}>
```

Comment:
Nit: we could extract this into an AppLike component to help future testing efforts.

Reply:
Yes please ;)